eucadeploy/plugins/debugger/debuggerplugin.py

```diff
@@ -28,35 +29,83 @@ def __del__(self):
         self.report()
 
     def success(self, message):
+        """
+        Function to display and tally success of a debug step
+        """
         self.passed +=1
         print green(self.message_style.format('DEBUG PASSED', message))
 
     def failure(self, message):
+        """
+        Function to display and tally a failure of a debug step
+        """
         self.failed +=1
         print red(self.message_style.format('DEBUG FAILED', message))
 
+    def info(self, message):
+        """
+        Function to display information of a debug step
```

Comment:
Can we make this single line comments like:

```# Function to display information of a debug step```

the other option is to make them proper docstrings, Pycharm can help here:
https://www.python.org/dev/peps/pep-0257/